.buildkite/scripts/run_standard_models.sh

```
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
```

I checked the blame, can't for the life of me quite remember what this comment is for lol I _think_ it had something to do with adding in this `-euo` flag for any failures to fail the run right away rather than continuing to execute. But we could probably just remove the comment or make it clearer

---

Sounds good and will remove in next commit!

---

Removed